ita-social-projects/ZeroWaste
# ZER-82 - Page for admin password change

dev
## JIRA

* [ZER-82 JIRA ticket](https://jira.softserve.academy/browse/ZER-82)


## Code reviewers

- [x] @olegsobchuk 

### Second Level Review

- [ ] @github_username

## Summary of issue

Add an Admin page for password changing

The new page should contain the following fields:
Old password
New password
New password confirmation
Apply necessary validations.

## Summary of change

Added page for admin password changing. Page contains fields for old password, new password, new password confirmation. Applied validations for password. Created actions for password change.

## Testing approach

1. Visit page [http://[::1]:3000/admins/admins/:admin_id/edit_password](http://[::1]:3000/admins/admins/:admin_id/edit_password) :admin_id - should be admin`s id
2. Fill in fields: current password, new password, confirm new password
3. Press the button "Change my password"

## CHECK LIST
